main.py

```diff
@@ -0,0 +1,9 @@
+num = int(input("enter any binary number"))
+sum = 0
+i = 0
+while num!=0:
+    rem = num % 10
+    sum = sum + rem * pow(2,i)
+    num = int(num/1o)

```

Comment:
```suggestion
    num = int(num/10)
```
Just do the correction by removing __'o'__ with 0.

Reply:
yaa done